From 43fe3c26a7b0b80cb35f644109c876f45c9e671a Mon Sep 17 00:00:00 2001
From: Jun Zhang <jun.zhang@arm.com>
Date: Wed, 15 Nov 2023 22:56:49 +0800
Subject: [PATCH] feat(rdkronos): Add FWU related configuration in rdkronos
 platform.mk

Enable PSA_FWU_SUPPORT, ARM_GPT_SUPPORT and set
NR_OF_IMAGES_IN_FW_BANK for rdkronos platform.

Upstream-Status: Pending
Signed-off-by: Jun Zhang <jun.zhang@arm.com>
---
 plat/arm/board/rdkronos/platform.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plat/arm/board/rdkronos/platform.mk b/plat/arm/board/rdkronos/platform.mk
index 91e7252d8..bab9e5633 100644
--- a/plat/arm/board/rdkronos/platform.mk
+++ b/plat/arm/board/rdkronos/platform.mk
@@ -17,6 +17,8 @@ override NEED_BL2U			:= 0
 override PSCI_EXTENDED_STATE_ID		:= 1
 override ENABLE_SVE_FOR_SWD		:= 1
 override ENABLE_SVE_FOR_NS		:= 1
+override ARM_GPT_SUPPORT		:=1
+override PSA_FWU_SUPPORT		:=1

 BL2_AT_EL3				:= 1
 CSS_USE_SCMI_SDS_DRIVER			:= 1
@@ -26,6 +28,7 @@ GICV3_SUPPORT_GIC600		        := 1
 HW_ASSISTED_COHERENCY			:= 1
 NEED_BL32				:= yes
 USE_COHERENT_MEM			:= 0
+NR_OF_IMAGES_IN_FW_BANK			:=1

 GIC_ENABLE_V4_EXTN	:= 1
 include drivers/arm/gic/v3/gicv3.mk
--
2.25.1

